pkg/cluster/k8sres_test.go

```diff
@@ -2993,6 +2993,44 @@ func TestGenerateResourceRequirements(t *testing.T) {
 				ResourceRequests: acidv1.ResourceDescription{CPU: k8sutil.StringToPointer("100m"), Memory: k8sutil.StringToPointer("100Mi")},
 			},
 		},
+		{
+			subTest: "test generation of resources when min limits are empty strings",
+			config: config.Config{
+				Resources: config.Resources{
+					ClusterLabels:        map[string]string{"application": "spilo"},
+					ClusterNameLabel:     clusterNameLabel,
+					DefaultCPURequest:    "",
+					DefaultCPULimit:      "",
+					MaxCPURequest:        "",
+					MinCPULimit:          "",
+					DefaultMemoryRequest: "",
+					DefaultMemoryLimit:   "",
+					MaxMemoryRequest:     "",
+					MinMemoryLimit:       "",
```

Comment:
You can change all these to "0", because "" is the internal default for string type in Golang. It is already tested in the subTest `test generation of resources when default is not defined`.